app/src/main/kotlin/com/intive/picover/main/view/MainActivity.kt

```diff
@@ -112,6 +113,7 @@ class MainActivity : ComponentActivity() {
 
 	private fun NavHostController.navigateWithSingleTop(item: NavigationItem) {
 		navigate(item.route) {
+			popUpTo(graph.findStartDestination().id)
```

Comment:
@michal-kucznerowicz Could describe what kind of behavior does it fix? Mentioned issue has only the description copied from the documentation but I'm not sure if anyone knows how it changes the navigation.

Reply:
For example, let's go to PartiesScreen then CameraScreen then PartiesScreen then CameraScreen.
Before this change, the stack looked like: A -> B -> A -> B
After this change, the stack will look like: A -> B
It's just simplified. There is no large stack of destinations.

Reply:
Nice, thank you 🍻